page-objects/Navigation.js

```diff
@@ -24,5 +24,7 @@ export class Navigation {
         await this.checkoutButton.waitFor();
         await this.checkoutButton.click();
         await this.page.waitForURL("/basket");
+        //added debuging code
+        console.log("Hello");
```

Comment:
```suggestion
        console.log("Hello");
```this piece of code is not debugging anything

Reply:
OK, I'll recheck